build/tokens/default.custom-properties.css

```diff
@@ -74,26 +74,44 @@
   --spacing-2: 2rem;
   --primary-xl-font-size-desktop: 58px;
   --primary-l-font-size-mobile: 34px;
-  --body-s-font-size-desktop: initial;
+  --secondary-m-font-size-desktop-xl: 14px;
+  --secondary-l-font-size-desktop-xl: 30px;
+  --body-s-font-size-desktop: 16px;
   --primary-s-font-size-desktop: 36px;
   --primary-xxs-font-size-desktop: 24px;
+  --secondary-regular-m-font-size-desktop-xl: 15px;
+  --accent-s-font-size-desktop-xl: initial;
```

Comment:
I'm thinking we don't want any values to be `initial` with the media query implementation...continuing review, just flagging this in case.